### PR TITLE
[docs] PR for Yugabyte Platform Multi-Zone universe document is out of date (Doc9499) 

### PR DESCRIPTION
**Pull request by lizayugabyte**
Preview: https://deploy-preview-10265--infallible-bardeen-164bc9.netlify.app/latest/yugabyte-platform/create-deployments/create-universe-multi-zone/

### CODE REVIEW

File: docs/content/latest/yugabyte-platform/create-deployments/create-universe-multi-zone.md [hunk 1]
```diff
@@ -110,7 +106,13 @@ You connect to a database node as follows:
 
 Yugabyte Platform includes a number of sample applications enclosed in Docker containers. 
 
-To access instruction on how to run sample applications, select your universe's **Overview** and then click **Actions > Run Sample Apps** to open the Run Sample Apps dialog shown in the following illustrations:
+The jdbc connection string is only needed if you're hooking up a Java application to the database. 
+
+The YSQL, YCQL, and Yedis endpoints are passed into the driver to let it know what IP addresses and ports to use to connect to the database. 
+
+The sample apps include the endpoint information in the command that is run on the platform node anyway.
+
```
Comment (ionthegeek) [inline docs/content/latest/yugabyte-platform/create-deployments/create-universe-multi-zone.md:114]: I recommend removing these lines. My comments were indented as feedback - this information isn't relevant to this section of the docs..

**Review by ionthegeek — CHANGES_REQUESTED** [review]
Some of my comments ended up being incorporated into the document. There are several lines I would recommend removing. I can approve after this change is made.

File: docs/content/latest/yugabyte-platform/create-deployments/create-universe-multi-zone.md (hunk 1 above)
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/create-deployments/create-universe-multi-zone.md:114]: @ionthegeek  Oh, sorry, these weren't meant to be in the doc. It was a copy-paste error. Thanks for catching it.

**Comment by lizayugabyte** [comment]
@ionthegeek I've just removed the content that was added in error. Please let me know if I can merge the PR.

**Review by ionthegeek — APPROVED** [review]

**Pull request merged**